### PR TITLE
Release v1.35.0

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix

/kind release

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Work on #3401 

**Special notes for your reviewer**:

### CODE REVIEW

**Review by markmandel — APPROVED** [review]

File: .gcloudignore [hunk 1]
```diff
@@ -11,19 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
```
Comment (markmandel) [inline .gcloudignore:14]: We don't need these changes anymore, since #3404 is working 👍🏻

File: build/includes/website.mk [hunk 2]
```diff
@@ -36,6 +36,8 @@ site-server: ensure-build-image
 site-static: ensure-build-image
 	-docker run --rm $(common_mounts) --workdir=$(mount_path)/site $(DOCKER_RUN_ARGS) $(build_tag) rm -r ./public
 	-mkdir $(agones_path)/site/public
+	docker run --rm $(common_mounts) --workdir=$(mount_path)/site $(DOCKER_RUN_ARGS) $(build_tag) \
+		bash -c "npx --version"
```
Comment (markmandel) [inline build/includes/website.mk:40]: I don't mind if we want to leave this in for future debugging, or delete it 🤷🏻 

Maybe delete to cleanup?

File: .gcloudignore (hunk 1 above)
Comment (Kalaiselvi84) [reply .gcloudignore]: I have reverted the change in .gcloudignore.

File: build/includes/website.mk (hunk 2 above)
Comment (Kalaiselvi84) [reply build/includes/website.mk]: I've removed the npx version check.

**Review by markmandel — APPROVED** [review]

**Pull request merged**